servers/visual/visual_server_scene.cpp

```diff
@@ -4059,6 +4307,7 @@ bool VisualServerScene::free(RID p_rid) {
 
 		instance_owner.free(p_rid);
 		memdelete(instance);
+		_interpolation_data.notify_free_instance(p_rid);
```

Comment:
these looks safe to notify after freeing the instance, but what if some one changes `notify_free_instance` in some unexpected fashion later on?  Should not `memdelete(instance);` be the last operation here?